students/12th/kyunghun/project_westagram/posting/views.py

```diff
@@ -0,0 +1,42 @@
+import json
+
+from django.views   import View
+from django.http    import JsonResponse
+
+from .models        import PostMedia, Photo
+from user.models    import Users
+    
+class PostingView(View):   
+    def post(self, request):
+            data                 = json.loads(request.body)
+            login_user           = Users.objects.get(email = data['email'])
+            login_user_id        = login_user.id
+            posting              = PostMedia.objects.create(
+                
+                title            = data['title'],
+                content          = data['content'],
+                user_id          = login_user_id
+            )
+            Photo.objects.create(
+                post_id          = posting.id,
+                image            = data['img']
+            )
+            return JsonResponse({'message':'SUCCESS'}, status=200)
+    
+    def get(self, request):
+        data                     = json.loads(request.body)
+        login_user               = Users.objects.get(email = data['email'])
+        login_user_id            = login_user.id
+        
+        posted_data              = PostMedia.objects.get(user_id= login_user_id)
+        posted_image             = Photo.objects.filter(post_id= posted_data.id)
+        
+        data_dict                = {'user_name':login_user.name}
+        image_list               = []
+        data_dict['title']       = posted_data.title
+        data_dict['content']     = posted_data.content
+
+        for img in posted_image:
+            image_list.append(img.image) 
```

Comment:
image_list가 선언되는 라인과 image_list에 데이터가 추가되는 라인은 붙어있는 것이 좋습니다 (코드를 읽는 흐름상)
data_dict를 선언하고 값을 추가하는 과정 사이에 image_list를 선언하는 부분이 중간에 끼어있는 것이 부자연스럽기 때문이기도 합니다.
```suggestion
        data_dict['title']       = posted_data.title
        data_dict['content']     = posted_data.content

        image_list               = []
        for img in posted_image:
            image_list.append(img.image) 
```